src/main/java/Parser/Parser.java

```diff
@@ -0,0 +1,72 @@
+package Parser;
+
+import Commands.*;
+import Exceptions.InvalidCommandException;
+import Exceptions.InvalidDescriptionException;
+import Tasks.Deadline;
+import Tasks.Event;
+import Tasks.Todo;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Parser {
+
+    /**
+     * Interprets the user's input and returns the respective command.
+     * @param command user's input string
+     * @return command according to user's input
+     * @throws InvalidDescriptionException empty description of task
+     * @throws InvalidCommandException unknown command
+     */
+    public static Command parse(String command) throws InvalidDescriptionException, InvalidCommandException {
```

Comment:
Your parse method seems a bit verbose, maybe you can employ a switch case instead of using if-else statements to make it clearer.

Also, you can consider abstracting out the cases into individual methods, such as to create a Todo object, to make it less verbose.